crates/runtime/src/accelerated_table/refresh_task_runner.rs

```diff
@@ -35,11 +35,9 @@ use super::refresh::Refresh;
 /// that only one [`RefreshTaskRunner`] is used per dataset, and that is is the only entity
 /// refreshing an `accelerator`.
 pub struct RefreshTaskRunner {
-    runtime_status: Arc<status::RuntimeStatus>,
     dataset_name: TableReference,
-    federated: Arc<dyn TableProvider>,
     refresh: Arc<RwLock<Refresh>>,
-    accelerator: Arc<dyn TableProvider>,
+    refresh_task: Arc<RefreshTask>,
```

Comment:
We need to keep a handle on the RefreshTask now to be able to update the inner AccelerationSink